releases/unreleased/add-support-for-outputting-main-dependencies.yml

```diff
@@ -0,0 +1,6 @@
+---
+title: Add support for outputting main dependencies
+category: added
+author: Lewi Uberg <lewi@uberg.me>
+issue: null
+notes: "Add possibility to output only non-dev dependencies to a txt file with the `-n, --non-dev` flag, and remove the default filename, forcing the user to provide one with the `-o, --output` flag. Usage: ``` $ peodd -n -o requirements.txt ```"
```

Comment:
Looks good but can you add the note in the multiple line format. You can do that by adding a `>` in the starting. Wrap the text to an approximate 72 characters or something.

```
notes: >
    Add possibility to output only non-dev dependencies 
    to a txt file with the `-n, --non-dev` flag, and 
    remove the default filename, forcing the user to 
    provide one with the `-o, --output` flag. 
    Usage: ``` $ peodd -n -o requirements.txt ```
```